src/plugins/saved_objects_management/public/management_section/register_section.tsx

```diff
@@ -0,0 +1,216 @@
+  setBreadcrumbs: (crumbs: ChromeBreadcrumb[]) => void;
+}) => {
+  const { service: serviceName, id } = useParams<{ service: string; id: string }>();
+  const capabilities = coreStart.application.capabilities;
+
+  const { search } = useLocation();
+  const query = parse(search);
+  const service = serviceRegistry.get(serviceName);
+
+  useEffect(() => {
+    setBreadcrumbs([
+      {
+        text: i18n.translate('savedObjectsManagement.breadcrumb.index', {
+          defaultMessage: 'Saved objects',
+        }),
+        href: '#/management/kibana/objects',
+      },
+      {
+        text: i18n.translate('savedObjectsManagement.breadcrumb.edit', {
+          defaultMessage: 'Edit {savedObjectType}',
+          values: { savedObjectType: service?.service.type ?? 'object' },
+        }),
+      },
+    ]);
+  }, [setBreadcrumbs, service]);
+
+  return (
+    <SavedObjectEdition
+      id={id}
+      serviceName={serviceName}
+      serviceRegistry={serviceRegistry}
+      savedObjectsClient={coreStart.savedObjects.client}
+      overlays={coreStart.overlays}
+      notifications={coreStart.notifications}
+      capabilities={capabilities}
+      notFoundType={query.notFound as string}
+    />
+  );
+};
+
+const SavedObjectsTablePage = ({
+  coreStart,
+  dataStart,
+  allowedTypes,
+  serviceRegistry,
+  actionRegistry,
+  setBreadcrumbs,
+}: {
+  coreStart: CoreStart;
+  dataStart: DataPublicPluginStart;
+  allowedTypes: string[];
+  serviceRegistry: ISavedObjectsManagementServiceRegistry;
+  actionRegistry: SavedObjectsManagementActionServiceStart;
+  setBreadcrumbs: (crumbs: ChromeBreadcrumb[]) => void;
+}) => {
+  const capabilities = coreStart.application.capabilities;
+  const itemsPerPage = coreStart.uiSettings.get<number>('savedObjects:perPage', 50);
+
+  useEffect(() => {
+    setBreadcrumbs([
+      {
+        text: i18n.translate('savedObjectsManagement.breadcrumb.index', {
+          defaultMessage: 'Saved objects',
+        }),
+        href: '#/management/kibana/objects',
+      },
+    ]);
+  }, [setBreadcrumbs]);
+
+  return (
+    <SavedObjectsTable
+      allowedTypes={allowedTypes}
+      serviceRegistry={serviceRegistry}
+      actionRegistry={actionRegistry}
+      savedObjectsClient={coreStart.savedObjects.client}
+      indexPatterns={dataStart.indexPatterns}
+      http={coreStart.http}
+      overlays={coreStart.overlays}
+      notifications={coreStart.notifications}
+      applications={coreStart.application}
+      perPageConfig={itemsPerPage}
+      goInspectObject={savedObject => {
+        const { editUrl } = savedObject.meta;
+        if (editUrl) {
+          // previously, kbnUrl.change(object.meta.editUrl); was used.
+          // using direct access to location.hash seems the only option for now,
+          // as using react-router-dom will prefix the url with the router's basename
+          // which should be ignored there.
+          window.location.hash = editUrl;
+        }
```

Comment:
So this is kinda ugly, however the `editUrl` property is actually the hash path of the `kibana` app it should redirect to. This is a big limitation, as it's based on the postulate that every inspectable types got their inspect url in the `kibana` app, however this can't be easily changed without changing the whole `SavedObjectsTypeManagementDefinition.getEditUrl` behavior, and I would really like to keep that refactoring for after every SO types registration has been migrated to NP/TS. 

Also see point 2 of previous comment regarding `RedirectToHomeIfUnauthorized`